lti_consumer/lti_1p3/extensions/rest_framework/authentication.py

```diff
@@ -0,0 +1,79 @@
+"""
+Django REST Framework extensions for LTI 1.3 & LTI Advantage implementation.
+
+Implements a custom authentication class to be used by LTI Advantage extensions.
+"""
+from django.utils.translation import ugettext as _
+from rest_framework import authentication
+from rest_framework import exceptions
+
+from lti_consumer.models import LtiConfiguration
+
+
+class Lti1p3ApiAuthentication(authentication.BaseAuthentication):
+    """
+    LTI 1.3 Token based authentication.
+
+    Clients should authenticate by passing the token key in the "Authorization".
+    LTI 1.3 expects a token like the following:
+        Authorization: Bearer jwt-token
+
+    Since the base implementation of this library uses JWT tokens, we expect
+    a RSA256 signed token that contains the allowed scopes.
+    """
+    keyword = 'Bearer'
```

Comment:
```suggestion
    KEYWORD = 'Bearer'
```

Reply:
I've based this class on https://github.com/encode/django-rest-framework/blob/master/rest_framework/authentication.py#L148 which uses a lowercase variable. I don't see any issue changing it to uppercase though.

Do you want me to change it?